process/charter.html

```diff
@@ -372,6 +370,13 @@ <h4>Updating the Charter End Date in the Charter</h4>
 (e.g., on the group's site) to say something like
 "Note: See changes from this initial schedule on the group home page."</li>
 </ul>
+	<li>When a group <strong>Chair</strong> (re)appointed or resigned, shortly before/after the announcements to w3c-ac-members@w3.org and cc's chairs@w3.org (sample emails [<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=appointed&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">1</a>][<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=steps+down&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">2</a>]), Team Contact (or Comm Team) modifies the charter, including:
```

Comment:
s/(re)appointed or resigned/is (re)appointed or resigns/

Reply:
```suggestion
	<li>When a group <strong>Chair</strong> (re)appointed or resigns, shortly before/after the announcements to w3c-ac-members@w3.org and cc's chairs@w3.org (sample emails [<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=appointed&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">1</a>][<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=steps+down&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">2</a>]), Team Contact (or Comm Team) modifies the charter, including:
```

Reply:
```suggestion
	<li>When a group <strong>Chair</strong> is (re)appointed or resigns, shortly before/after the announcements to w3c-ac-members@w3.org and cc's chairs@w3.org (sample emails [<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=appointed&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">1</a>][<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=steps+down&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">2</a>]), Team Contact (or Comm Team) modifies the charter, including:
```